cordformation/src/main/kotlin/net/corda/plugins/Cordformation.kt

```diff
@@ -37,18 +37,23 @@ class Cordformation : Plugin<Project> {
             return outputFile
         }
 
+        fun createJarRegex(jarName: String, releaseVersion: String): Regex {
+            return "\\Q$jarName\\E(-enterprise)?(-jdk11)?-\\Q$releaseVersion\\E(-.+)?\\.jar\$".toRegex()
```

Comment:
Could we make it so that it's not jdk11 specific, maybe something like jdk[0-9]+ ?

Reply:
Do you expect a Java 17 version of Corda 4?

Reply:
It was just hardwiring a jdk version that seemed a bit strange. I guess we can update the plug-in if we need to support anything other than jdk11

Reply:
I didn't want to claim for something that we would never create, and as far as I know that's currently just Java 11.

The 5.1.x branch has replaced this regular expression entirely because it has created an explicit `corda` configuration to seach for `corda.jar` inside. (And possibly `corda-testserver` / `corda-webserver` too, but that's all!)